### PR TITLE
V10.5.0 Changelog + Resolve conflicts with master branch

### PR DESCRIPTION
**Pull request by zezha-msft**

### CODE REVIEW

File: ChangeLog.md [hunk 1]
```diff
@@ -5,17 +5,35 @@
...
+Doing so allows the names to not be an alphabetic series, which used to negatively impact the performance on the service side.
+
+### Bug fixes
 
+1. Fixed the SAS timestamp parsing issue.
+1. Transfers to the File Service with a read-only SAS were failing because we try listing properties for the parent directories.
+The user experience is improved by ignoring this benign error and try creating parent directories directly.
+1. Fixed issue with mixed SAS and AD authentication in the sync command.
+1. Fixed file creation error on Linux when decompression is turned on.
+1. Fixed issue for files with "%00" in their names.
```
Comment (mohsha-msft) [inline ChangeLog.md:34]: Fixed issue for files with extended charset such as [%00 - %19, %0A-%0F, %1A-%1F]

File: ChangeLog.md [hunk 2]
```diff
@@ -5,17 +5,35 @@
...
+1. Improved the naming of files being generated in benchmark command, by reversing the digits. 
+Doing so allows the names to not be an alphabetic series, which used to negatively impact the performance on the service side.
+
+### Bug fixes
 
+1. Fixed the SAS timestamp parsing issue.
+1. Transfers to the File Service with a read-only SAS were failing because we try listing properties for the parent directories.
+The user experience is improved by ignoring this benign error and try creating parent directories directly.
+1. Fixed issue with mixed SAS and AD authentication in the sync command.
+1. Fixed file creation error on Linux when decompression is turned on.
```
Comment (nakulkar-msft) [inline ChangeLog.md:33]: Missing Linux Decompressing writes bug - #1029

File: ChangeLog.md (hunk 1 above)
Comment (mohsha-msft) [reply ChangeLog.md:34]: + Fixed issue in which attribute filters does not work if source path contains an asterisk in it.
+ Fixed issue of unexpected upload destination when uploading a whole drive in Windows. eg. "D:\\"

File: ChangeLog.md (hunk 2 above)
Comment (zezha-msft) [reply ChangeLog.md:33]: It's already on line 33, right?

**Pull request merged**